default/cve5/script.js

```diff
@@ -560,3 +560,79 @@ function cveFixForVulnogram(j) {
     }
     return j;
 }
+
+let previousVersions = null;
+
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
please define the watchers in the schema file itself and not here.
please see how CVSS scores are set.